Dynamic data structures/Advanced bracket balance/main.c

```diff
@@ -0,0 +1,58 @@
+﻿#include <stdio.h>
+#include <locale.h>
+#include <stdlib.h>
+
+#include "stack.h"
+#include "advancedBracketBalance.h"
+#include "userInput.h"
+#include "testsForAdvancedBracketBalance.h"
+#include "testsForStack.h"
+
+void runningTests(bool* errorCode) {
```

Comment:
```suggestion
void runTests(bool* errorCode) {
```
В повелительной форме должно быть